### ISSUE TITLE
sideEffects: false causes webpack to strip css import

### ISSUE THREAD

**Issue by lroling8350** [GetStream/stream-chat-react#934]
In version 5.4.0+ the package.json added the value

```
"sideEffects": false
```

This causes webpack to evaluate the css import as shakeable and removed from a production build

```
import 'stream-chat-react/dist/css/index.css';
```

After reading this section on [tree shaking](https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free) it seems this can be addressed by updating the package.json to

```
sideEffects: ["*.css"] 
```

For now we are simply changing the import to 
```
import * as streamStyles from 'stream-chat-react/dist/css/index.css';
```

and adding a hidden div with one of the styles to prevent webpack from tree shaking the css and will update once a fix is in place.

**Comment by DanC5** [comment]
@lroling8350 good catch here. I was actually planning to make a patch release today anyways, so will look into this and roll it out. Just out of curiosity, do you have any of your own css files in your app? The build process didn't shake the css out of the other apps we've tested, so I'm wondering what's different in your setup that would purge the css.
